lib/src/pubspec.dart

```diff
@@ -245,82 +243,6 @@ class Pubspec {
 
   Map<String, Feature> _features;
 
-  /// The configurations of the transformers to use for this package.
-  List<Set<TransformerConfig>> get transformers {
```

Comment:
It would be good to add validators that produce warnings when trying to publish packages that include a `transformers` field or a `web.compiler` field.